src/index.js

```diff
@@ -0,0 +1,125 @@
+  const key = 'api_key=574dd9cb25756d0a612a187ad048b948';
+  const urrl = `https://api.themoviedb.org/3/discover/movie?sort_by=popularity.desc&${key}`;
+  const response = await fetch(urrl);
+  const datax = await response.json();
+  const resp = await datax.results;
+  return resp;
+};
+
+function displayMov() {
+  newData().then((movies) => {
+    movies.forEach((movie, index) => {
+      const containing = document.querySelectorAll('.comment-btn')[index];
+      containing.addEventListener('click', (event) => {
+        if (event.target.matches('button')) {
+          popup(movie);
+          displayCommentinfo(movie, receiveComment, counter, popup);
+        }
+      });
+    });
+  });
+}
+
+function displayReservation() {
+  newData().then((movies) => {
+    movies.forEach((movie, index) => {
+      const Select_R_btn = document.querySelectorAll('.reservation-btn')[index];
+      
+      Select_R_btn.addEventListener('click', (event) => {
+        if (event.target.matches('button')) {
+          Res_popup(movie);
+          displayReservationinfo(movie, receiveReservation, resCounter);
+        }
+      });
+    });
+  });
+}
+
+const getData = async () => {
+  const apiKey = 'api_key=574dd9cb25756d0a612a187ad048b948';
+  const apiUrl = `https://api.themoviedb.org/3/discover/movie?sort_by=popularity.desc&${apiKey}`;
+  const response = await fetch(apiUrl);
+  const data = await response.json();
+  const res = await data.results;
+  showMovies(res);
+  displayMov(res);
+  displayReservation(res);
+  newData().then((movies) => {
+    countermov(movies, showCounter);
+    })  
+};
+
+getData();
+
+async function postData(url = '', data = {}) {
+  const response = await fetch(url, {
+    method: 'POST',
+    mode: 'cors', 
+    cache: 'no-cache',
+    credentials: 'same-origin',
+    headers: {
+      'Content-Type': 'application/json'
+    },
+    redirect: 'follow', 
+    referrerPolicy: 'no-referrer', 
+    body: JSON.stringify(data)
+  });
+   response.json(); 
+   window.location.reload();
+}
+
+async function getText(file) {
+  let myObject =  await fetch(file);
+  let myText = await myObject.json();
+  likes(myText);
+  return myText;
+}
+
+const likes = ((alfa) => {
+  newData ().then((movies) => {
+    movies.forEach((movi, index) => {
+  const spans = document.querySelectorAll('.like-count')[index];
+  console.log(alfa);
+      alfa.forEach(lik => {
+        if(lik.item_id == movi.id) {
+          spans.innerHTML = `${lik.likes} likes`;
+    
+      };
+    })
+    });
+   
+});
+});
+    
+getText(`https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/fVSALNrpnLiYk8wyVjPw/likes`);
+
+ 
+  newData ().then((movies) => {
+    movies.forEach((movi, index) => {
+      const spans = document.querySelectorAll('.cards')[index];
+      spans.addEventListener('click', (e) => { 
+  if (e.target.tagName.toLowerCase() === 'i'){
+    console.log(movi.id);
+    const itemId = movi.id; 
+    console.log(itemId);
```

Comment:
- [ ] Kindly remove the console statement as they may cause security concerns by exposing data to the user.

Reply:
Addressed 👍🏼